lib/winston/container.js

```diff
@@ -40,7 +40,11 @@ module.exports = class Container {
 
       // Remark: Make sure if we have an array of transports we slice it to
       // make copies of those references.
-      options.transports = existing ? existing.slice() : [];
+      if(existing) {
+        options.transport = Array.isArray(existing) ? existing.slice() : [existing];
```

Comment:
Should this be `options.transports` not `transport` (same comment below)?

Reply:
Fixed it :see_no_evil: